agent/src/main/java/org/openremote/agent/protocol/dmx/AbstractArtnetLightState.java

```diff
@@ -0,0 +1,19 @@
+package org.openremote.agent.protocol.dmx;
```

Comment:
Don't understand why some files are in the `dmx` package whilst others are in `dmx.artnet`